lms/envs/production.py

```diff
@@ -470,13 +470,18 @@ def get_env_setting(setting):
 AWS_QUERYSTRING_AUTH = AUTH_TOKENS.get('AWS_QUERYSTRING_AUTH', True)
 AWS_S3_CUSTOM_DOMAIN = AUTH_TOKENS.get('AWS_S3_CUSTOM_DOMAIN', 'edxuploads.s3.amazonaws.com')
 
+# this feature introduced in django-storages=1.13.2
+# for details https://github.com/openedx/edx-platform/pull/33301
+AWS_S3_USE_THREADS = False
```

Comment:
https://github.com/jschneier/django-storages/pull/1112/files#diff-421acfde5179532e21813bca0e6b91a717c16d3628f57b5a77c55e3afba6862fR336

Reply:
its a new variable and its default is True in django-storages.

Reply:
Previously, storages was not setting any value for `use_threads`, so the default was true.

https://github.com/boto/boto3/blob/1.7.0/boto3/s3/inject.py#L425
https://github.com/boto/boto3/blob/1.7.0/boto3/s3/transfer.py#L176C1-L177C1

Reply:
good catch